src/features/deleteUser/ui.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button, DeleteForeverIcon } from 'shared';
+import { viewerModel } from 'entities/viewer';
+import { deleteUserModel } from 'features/deleteUser';
+
+export const DeleteUser: React.FunctionComponent = () => {
+  return (
+    <Button
+      onClick={() => {
+        deleteUserModel.userDeleted();
```

Comment:
don't keep logic in UI, call viewerModel from store

Reply:
what do you mean? this is the event from model

Reply:
yeah, and it should not be here

Reply:
fixed, if i understand right